securedrop/debian/app-code/lib/systemd/system/securedrop-remove-pending-sources.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=job to remove pending SecureDrop sources daily
+
+[Service]
+ExecStart=/var/www/securedrop/manage.py remove-pending-sources >/dev/null
```

Comment:
I don't think we should be suppressing output like this. If the goal is to prevent OSSEC notifications (which the PR description alludes to), I don't think it should be necessary, other timers/services generate output but don't trigger notifications?

If it's producing output that shouldn't be logged, then I'd rather have a --quiet flag or something.

Reply:
You know what: we can retitle this pull request "In which @cfm writes a systemd timer exactly _as though_ it is a cron job".  :-P

The suppression was just for OSSEC, so you're right that it's superfluous as well as non-idiomatic.  Removed in 6f780c6.

Reply:
Haha, lgtm, leaving the rest of the review for Nathan :)